src/pages/Root.tsx

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+import { Outlet } from 'react-router-dom';
+import { ThemeProvider } from '@mui/material';
+import theme from '../assets/styles/theme';
+
+const RootLayout: React.FC = () => {
+  return (
+    <ThemeProvider theme={theme}>
+      <main>
```

Comment:
The <main> tag isn't strictly needed, but I thought it may be a good practice to indicate the primary content, especially since the theme is applied across all pages. Should I delete it?

Reply:
But what is primary content in this case? The code indicates it is the entire page and every page. It makes more sense for primary content to be a specific part of a page.